js/foam/u2/Element.js

```diff
@@ -305,7 +306,49 @@ CLASS({
       name: 'innerHTML',
       transient: true,
       getter: function() { return this.outputInnerHTML(this.createOutputStream()).toString(); }
-    }
+    },
+    {
+      name: 'minWidth',
```

Comment:
Are these the sorts of things we should be adding to element?  It feels out of place, this is not used on all elements, only in a few context.

What about putting these in a trait only applied to views that are intended to be used in the stackview or something.